swagger-config.yaml

```diff
@@ -201,9 +201,54 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/StatesResult'
+  /trends:
+    get:
+      tags:
+        - Trends
+      summary: "List trends"
+      description: "Return specific aggregations for a search"
+      parameters:
+        - $ref: '#/components/parameters/search_term'
+        - $ref: '#/components/parameters/field'
+        - $ref: '#/components/parameters/size'
```

Comment:
Do we use size for trends?  I think since it is an aggregation we don't need/use it

Reply:
We do use size in the aggregations, which is why I kept it in at first. However on second look we actually use `trend_depth` for the size param in the aggregation requests. I've removed `size`